index.js

```diff
@@ -333,7 +366,7 @@ export default class ImageView extends Component<PropsType> {
     }
 
     onNextImagesReceived(images: Array<ImageType>, imageIndex: number = 0) {
-        this.imageInitialParams = images.map(getInitalParams);
+        this.imageInitialParams = images.map(getInitalParams.bind(null, this.state.screenDimensions));
```

Comment:
Please, retrieve `screenDimensions` from `this.state` in const variable and use it farther in this method

Reply:
*this.state* is not reachable in const function